### DISCUSSION TITLE
Encryption version for your account is outdated and requires upgrade.

### DISCUSSION THREAD

**Discussion by git5001** [standardnotes/forum#3113] [General]
**Describe the bug**
Logging into any existing account gives the following error dialog (with the latest desktop -3.161 version)

_Update Recommended
The encryption version for your account is outdated and requires upgrade. You may proceed with login, but are advised to perform a security update using the web or desktop application. If your account was created after 1.577.836.800.000, it may not be safe to continue signing in. In that case, please discontinue your sign in request and contact support. For more information, visit standardnotes.com/help/security._

Using sign in in this dialog does nothing and jumps back. Using Cancel also does the same.

A new account worked once on creation but after logout a second login is again not possible

**To Reproduce**
1. Create a new account and sign out and sign in again
2. Might or might not need self hosted server (unclear)

**Expected behavior**

- Describe in the error dialogue or the referenced webpage what to do to fix the error
- or  allow to continue the program

**Desktop (please complete the following information):**
 - OS: Windows 10 or 11 

**Additional context**
This is with a self hosted server. The error occurred with an older server version but the same with an updated version from today.

This behaviour does not occur with a very old version of the app.

**Comment by effieeee** [comment]
Hi @git5001, this will require some troubleshooting over emails, so please contact us at help@standardnotes.com using your account email so that we can continue assisting you with this 🙏
